include/deal.II/base/vectorization.h

```diff
@@ -716,6 +724,33 @@ vectorized_load_and_transpose(const unsigned int              n_entries,
 }
 
 
+/**
+ * The same as above with the difference that an array of width pointers are
```

Comment:
```suggestion
 * The same as above with the difference that an array of pointers are
```
I think `width` is clear from the argument.